sycl/test-e2e/ESIMD/unified_memory_api/Inputs/atomic_update.hpp

```diff
@@ -260,7 +260,7 @@ bool test_acc(queue q, const Config &cfg) {
     return true;
   } else {
     // Remove once 1 and 2 are supported
-    if constexpr (n_args != 0)
+    if constexpr (n_args == 2)
       return true;
```

Comment:
`if constexpr` doesn't work same way as regular `if`.
compiler will compile the rest of the code anyways.
for constexor it is better to use
if constexpr (condition) { X; } else { Y; }